zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -1094,4 +1212,38 @@ mod tests {
             test_bytecode_circuit_unrolled::<Fr>(k, vec![invalid], false);
         }
     }
+
+    use halo2_proofs::dev::MockProver;
+
+    #[test]
+    fn bytecode_soundness_bug_1() {
```

Comment:
```suggestion
    #[test]
    #[should_panic]
    fn bytecode_soundness_bug_1() {
```

See https://doc.rust-lang.org/book/ch11-01-writing-tests.html#checking-for-panics-with-should_panic

This allows us to keep this negative test and still pass the CI